### PR TITLE
fixed travis timeout for php 5.5 test case

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

Debug php 5.5 on travis

#### Why?

Currently composer install fails on php 5.5

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ matrix:
   include:
     - php: 5.5
       env:
-        - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
+        - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction -vvv"
```
Comment (alexander-schranz) [inline .travis.yml]: this seems the only way that travis end up with 10m without output error 🙈

Comment (wachterjohannes) [reply .travis.yml]: dont like it 🙊 @danrot what do you think?

Comment (alexander-schranz) [reply .travis.yml]: @wachterjohannes I also don't like it but its the only way composer does a output and travis does not say there is no 10m output

**Comment by danrot** [comment]
This should go into the 1.5 branch.

**Comment by alexander-schranz** [comment]
@danrot done

**Pull request merged**